gemrb/plugins/WEDImporter/WEDImporter.cpp

```diff
@@ -372,6 +372,9 @@ void WEDImporter::ReadWallPolygons()
 			if (flags&WF_BASELINE) {
 				polygonTable[i]->SetBaseline(base0, base1);
 			}
+			if (core->HasFeature(GF_PST_STATE_FLAGS)) {
```

Comment:
Not sure if this is the correct way to check for PST here.

Reply:
Ideally it would be a new feature flag, but if we were strict, we'd have too many already. Another candidate is GF_AREA_OVERRIDE, but it's not any better, so let's just leave it.